paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetLoader.kt

```diff
@@ -62,18 +62,7 @@ internal class DefaultCustomerSheetLoader(
 
     override suspend fun load(configuration: CustomerSheet.Configuration): Result<CustomerSheetState.Full> {
         return runCatching {
-            val customerAdapter = customerAdapterProvider.awaitAsResult(
-                timeout = 5.seconds,
-                error = {
-                    "Couldn't find an instance of CustomerAdapter. " +
-                        "Are you instantiating CustomerSheet unconditionally in your app?"
-                },
-            ).onFailure {
-                errorReporter.report(
-                    errorEvent = ErrorReporter.ExpectedErrorEvent.CUSTOMER_SHEET_ADAPTER_NOT_FOUND,
-                    stripeException = StripeException.create(it)
-                )
-            }.getOrThrow()
+            val customerAdapter = retrieveCustomerAdapter().getOrThrow()
 
             val elementsSessionWithMetadata = retrieveElementsSession(
                 configuration = configuration,
```

Comment:
There are a couple more failure reports done here. Going to break them out into their own functions once the logic in them is moved around.